pkg/controller/storagecluster/reconcile.go

```diff
@@ -924,6 +926,15 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, nodeCount int, r
 	} else {
 		reqLogger.Info(fmt.Sprintf("No monDataDirHostPath, monPVCTemplate or storageDeviceSets configured for storageCluster %s", sc.GetName()))
 	}
+	multusNetwork := sc.Spec.Network
```

Comment:
What about adding a method to the spec to validate whether networking is enabled?
So you do something like `if sc.Spec.Network.IsMultus()` and then you enter your condition.

Reply:
done